eventorder_test.go

```diff
@@ -0,0 +1,403 @@
+//go:build integration
+
+package main_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
```

Comment:
This package is deprecated we should use `io` instead, I am wondering why linters don't complain